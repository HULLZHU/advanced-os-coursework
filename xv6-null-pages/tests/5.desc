testing good arguments for mprotect system call