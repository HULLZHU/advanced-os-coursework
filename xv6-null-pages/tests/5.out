XV6_TEST_OUTPUT: Return value of mprotect with len 1: 0
XV6_TEST_OUTPUT: Return value of mprotect with len 2: 0
